Source/WebCore/rendering/StyledMarkedText.cpp

```diff
@@ -180,36 +178,37 @@ static Vector<StyledMarkedText> coalesceAdjacentWithSameRanges(Vector<StyledMark
     ASSERT(!styledTexts.isEmpty());
     Vector<StyledMarkedText> frontmostMarkedTexts;
     frontmostMarkedTexts.append(styledTexts[0]);
-    for (auto it = styledTexts.begin() + 1, end = styledTexts.end(); it != end; ++it) {
-        StyledMarkedText& previousStyledMarkedText = frontmostMarkedTexts.last();
+    for (size_t i = 1; i < styledTexts.size(); ++i) {
```

Comment:
This can be a ranged for loop.

Reply:
How? I would need a boolean variable to skip the first item I think? If so, it didn't feel better to me. Is there a better way to write this?

Reply:
Oops! I didn't notice the 1. Yeah, I think you're right.